### PR TITLE
#85463 Value resolver for Zaak data

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: using-valtimo/forms/creating-forms-in-valtimo.md [hunk 1]
```diff
@@ -22,6 +22,44 @@ property `targetKey` is added, it will store submitted values to the reference p
...
+- `zaakstatus` (Zaken API & Catalogi API)
+- `zaakobject` (Zaken API)
+
+With the `zaak` prefix, all data inside
+a [zaak response](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/zaken-api/master/src/openapi.yaml#tag/zaken/operation/zaak_create)
+can be used to prefill the form. Some examples:
+
+| Source key         | Type of data                                            | Example                                                                         |
+|--------------------|:--------------------------------------------------------|---------------------------------------------------------------------------------|
+| zaak:url           | A URL reference to the zaak                             | https://example.com/zaken/api/v1/zaken/f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06     |
```
Comment (marijnritense) [inline using-valtimo/forms/creating-forms-in-valtimo.md:40]: I think these tables should be under references instead

Comment (Klaas-Ritense) [reply using-valtimo/forms/creating-forms-in-valtimo.md]: Moved

File: reference/modules/form.md [hunk 2]
```diff
@@ -36,6 +38,41 @@ instances for the case.
...
+
+With the `zaak` prefix, all data inside
+a [zaak response](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/zaken-api/master/src/openapi.yaml#tag/zaken/operation/zaak_create)
+can be used to prefill the form. Some examples:
+
+| Source key         | Type of data                                            | Example                                                                         |
+|--------------------|:--------------------------------------------------------|---------------------------------------------------------------------------------|
+| zaak:url           | A URL reference to the zaak                             | https://example.com/zaken/api/v1/zaken/f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06     |
+| zaak:uuid          | Zaak UUID                                               | f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06                                            |
+| zaak:identificatie | The URL to the zaak                                     | ZK2023-00001                                                                    |
```
Comment (ivo-ritense) [inline reference/modules/form.md:55]: "The URL to the zaak" 

This isn't correect is it?

Comment (Klaas-Ritense) [reply reference/modules/form.md]: Changed it

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**